docs/source/en/_toctree.yml

```
@@ -162,6 +162,8 @@
       title: AutoencoderKL
     - local: api/models/asymmetricautoencoderkl
       title: AsymmetricAutoencoderKL
+    - local: api/models/autoencoder_tiny
+      title: Tiny Autoencoder
```

To stay consistent with how its spelled in the original repository.

```suggestion
      title: Tiny AutoEncoder
```